### DISCUSSION TITLE
scroll view up to the top when selecting a field type in the drawer that's near the bottom of screen

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13430] [Feature Requests]
When selecting a field type in the drawer, if you select one near the bottom of the screen, it'd be nice if it scrolled the top of the options to the top of the screen to improve usability and prevent having to scroll down.

There may be other places where this could be applied!

**Comment by rijkvanzanten** [comment]
The private-view is currently forcing the scroll back up to 0 for every navigation:

https://github.com/directus/directus/blob/d46886d5ac184449f6c8ab5129ae023420d2b129/app/src/views/private/private-view.vue#L148-L151

For most page transitions, this is the desired behavior (as it'd be very annoying to start halfway down the form when navigating from a layout to a detail page for example). That being said, this could definitely be improved by adding a list of pages that we don't want this logic to apply to (like the data-model fields page)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
